src/core/image.js

```diff
@@ -506,6 +506,122 @@ var PDFImage = (function PDFImageClosure() {
       }
     },
 
+    /**
+     * Check whether resize image data or not
+     * @param {Number} comps Number of color components, 1 or 3 is supported.
```

Comment:
This comment doesn't seem to be equivalent to the code, since there is no `comps === 3` case in the code.